tests/test_earthspy.py

```diff
@@ -330,6 +330,20 @@ def test_sentinelhub_request(self) -> None:
         # # check that a Sentinel Hub request was created
         assert isinstance(sr2, shb.SentinelHubRequest)
 
+    def test_geojson(self) -> None:
+        """Test geojson files"""
+
+        for file in os.listdir("earthspy/data"):
```

Comment:
I'd rather use `glob.glob` here, and maybe put that list in in a variable before the loop.